### ISSUE TITLE
Depwarn caused by non-deprecated entry point `exe`

### ISSUE THREAD

**Issue by IanButterworth** [JuliaIO/FFMPEG.jl#57]
With `--depwarn=yes`
```
julia> FFMPEG.exe("-version")
┌ Warning: ffmpeg() is deprecated, use the non-do-block form
│   caller = ip:0x0
└ @ Core :-1
...
```

With `--depwarn=error`
```
julia> FFMPEG.exe("-version")
ERROR: ffmpeg() is deprecated, use the non-do-block form
Stacktrace:
 [1] depwarn(msg::String, funcsym::String; force::Bool)
   @ Base ./deprecated.jl:124
 [2] depwarn(msg::String, funcsym::String)
   @ Base ./deprecated.jl:121
 [3] ffmpeg(f::Function; adjust_PATH::Bool, adjust_LIBPATH::Bool)
   @ FFMPEG_jll ~/.julia/packages/JLLWrappers/pG9bm/src/products/executable_generators.jl:20
 [4] ffmpeg(f::Function)
   @ FFMPEG_jll ~/.julia/packages/JLLWrappers/pG9bm/src/products/executable_generators.jl:19
 [5] #exe#2
   @ ~/.julia/packages/FFMPEG/OUpap/src/FFMPEG.jl:0 [inlined]
 [6] exe(args::String; command::Function, collect::Bool)
   @ FFMPEG ~/.julia/packages/FFMPEG/OUpap/src/FFMPEG.jl:64
 [7] top-level scope
   @ REPL[9]:1
```

Also, I think this is more confusing than it needs to be https://github.com/JuliaPackaging/JLLWrappers.jl/issues/59

**Comment by cormullion** [comment]
Yes, I'm wondering how to get rid of this message too:

```plain
┌ Warning: ffmpeg() is deprecated, use the non-do-block form
│   caller = ip:0x0
└ @ Core :-1
```

using  FFMPEG.jl 0.4.1.

**Comment by giordano** [comment]
I believe that's already fixed by #47, but there haven't been new releases for over 3 years.

**Comment by IanButterworth** [comment]
Closed as we just made a new release
